feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/balance/list/WalletScreen.kt

```diff
@@ -74,21 +76,28 @@ fun WalletScreen(
     val scope = rememberCoroutineScope()
     val keyboardController = LocalSoftwareKeyboardController.current
 
+    val screenCallback = object : ChainSelectContentInterface {
```

Comment:
Same here, just implement ChainSelectContentInterface in viewModel and pass it as callback